### PR TITLE
HTBHF-2167 Added my address is not listed link.

### PR DESCRIPTION
**Pull request by dwybourn**
- Added a link to the manual address page.
- Reordered the manual address and postcode lookup pages as we can not link to a future step.

### CODE REVIEW

File: src/web/views/select-address.njk [hunk 1]
```diff
@@ -35,8 +35,11 @@
                 },
                 items: addresses
             }) }}
+
+            <a class="govuk-link govuk-!-font-size-19" href="/manual-address">{{ addressNotListed }}</a>
         {% endif %}
 
+
```
Comment (gethyn1) [inline src/web/views/select-address.njk]: Rogue new line

Comment (dwybourn) [reply src/web/views/select-address.njk]: Fixed.

File: src/test/a11y/tests/apply.js [hunk 2]
```diff
@@ -65,9 +65,19 @@ const apply = [
     })
   },
   {
-    url: URLS['POSTCODE'],
+    url: URLS['MANUAL_ADDRESS'],
```
Comment (YetAnotherMatt) [inline src/test/a11y/tests/apply.js]: Why have we moved manual address before postcode lookup? Not that it makes any functional difference at this point, but in terms of page flow, you should only be able to get to manual address after postcode lookup...

File: src/test/common/steps/navigation/step-page-actions.js [hunk 3]
```diff
@@ -48,6 +48,11 @@ const STEP_PAGE_ACTIONS = [
     page: (pages) => pages.nationalInsuranceNumber,
     actions: async () => enterNinoAndSubmit()
   },
+  {
```
Comment (YetAnotherMatt) [inline src/test/common/steps/navigation/step-page-actions.js]: Same question again here - why is manual address before postcode lookup?

File: src/test/a11y/tests/apply.js (hunk 2 above)
Comment (dwybourn) [reply src/test/a11y/tests/apply.js]: Because you can't link to a page further in the flow. Our state machine will redirect you.

**Pull request closed without merging**